Sources/PackageModelSyntax/AddTarget.swift

```diff
@@ -184,14 +245,35 @@ public struct AddTarget {
             """
 
         case .test:
-            """
-            \(imports)
-            class \(raw: target.name): XCTestCase {
-                func test\(raw: target.name)() {
-                    XCTAssertEqual(42, 17 + 25)
+            switch configuration.testHarness {
+            case .none:
+                """
+                \(imports)
+                // Test code here
+                """
+
+            case .xctest:
+                """
+                \(imports)
+                class \(raw: target.name): XCTestCase {
+                    func test\(raw: target.name)() {
+                        XCTAssertEqual(42, 17 + 25)
+                    }
+                }
+                """
+
+            case .swiftTesting:
+                """
+                \(imports)
+                @Suite
+                struct \(raw: target.name)Tests {
+                    @Test("\(raw: target.name) tests")
+                    func tests() {
```

Comment:
Stylistic suggestion: I recommend naming the func something not prefixed by the word "test", like `example()`. The `@Test` attribute already marks it as a test so having that prefix can feel redundant. (I realize this was just adapted from the prior XCTest template, of course!)

Reply:
Sure, I'll drop the test prefix for swift-testing.